static/javascript/launch.js

```diff
@@ -354,9 +358,12 @@ function getInputField(
   const value = defaultValue ? `value="${defaultValue}"` : "";
   const readOnly = isReadOnly ? "readonly" : "";
   if (readOnly) {
-    return `<input ${readOnly} id="${fieldName}" type="${type}" ${value} class="qa-${fieldName}" onchange="${onChangeCallback}">`;
+    return `<input ${readOnly} id="${fieldName}" type="${type}" ${value} class="qa-${fieldName} ons-input ons-input--text" onchange="${onChangeCallback}">`;
```

Comment:
Are these "qa" classes doing anything? Ive searched the repo and they only seem to be set in some places in the js but not actually used for anything

Reply:
I've had a look now and the places they are referred to in JS is just to inject HTML back into the code, maybe they can be removed?

Reply:
Yeah, remove them and give it a good test

Reply:
updated 👍